package.json

```diff
@@ -18,21 +18,21 @@
     "subscribableevent": "^1.0.1"
   },
   "peerDependencies": {
-    "react": "^16.0",
-    "react-dom": "^16.0",
+    "react": "^16.3",
+    "react-dom": "^16.3",
     "react-native": ">=0.57 <0.60",
```

Comment:
My only question about this PR (and the release path to 2.x) is why not loose the react-native dependency band to 0.60+ (I'm using it on 0.61 without issue and don't see anything in 0.62 that will break it now that netinfo and webview are standing up nicely with current lean-core extracted repos)

I see the peerDep set to <0.60 for the main module here and all the extensions and it seems like noise in the package install process when they seemingly work fine up to 0.61 at minimum

This is a separate statement from "the samples should be upgraded to RN0.6x" as that's a not fun chore but is orthogonal from the peerDependency setting I think

Reply:
Good question... I coulda sworn there was some work left to do to make it compatible with 0.60+ but maybe it was just the samples that needed the chore and we're good.

Reply:
This was my list as I brought my work project up on it - seemed like it was just non-blocking things now (React lifecycle items etc) https://github.com/microsoft/reactxp/issues/1123#issuecomment-520096927